src/TableV8/stories.tsx

```diff
@@ -17,15 +17,37 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
stories are WIP - I'll improve them if everything else is OK

Reply:
Follow up to Slack convo, we should make story follow up tickets to pick up when time allows